### PR TITLE
Better "inheritance search disabled" experience

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

**Comment by hanars** [comment]
passing tests: https://travis-ci.org/github/broadinstitute/seqr/builds/756476382

File: ui/pages/Search/components/filters/CustomInheritanceFilter.jsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ const CustomInheritanceFilter = React.memo(({ value, onChange, family, individua
     <Table basic="very" compact>
       <Table.Body>
         {individuals.map((individual, i) =>
-          <Table.Row key={individual.individualGuid}>
+          <Table.Row key={individual.individualGuid} disabled={!individual.sampleGuids.length}>
```
Comment (ShifaSZ) [inline ui/pages/Search/components/filters/CustomInheritanceFilter.jsx:58]: Simply disable the row with no data without telling the reason maybe is not user-friendly.

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: ui/pages/Search/components/filters/CustomInheritanceFilter.jsx (hunk 1 above)
Comment (hanars) [reply ui/pages/Search/components/filters/CustomInheritanceFilter.jsx:58]: fair, I'll add a hover text

Comment (hanars) [reply ui/pages/Search/components/filters/CustomInheritanceFilter.jsx]: added

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**